app/views/admin/performances/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<% content_for :page_title, 'Performance' %>
```

Comment:
Might want to add something like `<% content_for :breadcrumb_leaf, 'Performance' %>` so that this has the correct breadcrumb?

![Screenshot 2019-05-02 at 14 22 15](https://user-images.githubusercontent.com/282788/57079176-bca9ca80-6ce7-11e9-957d-ab57ecb21607.png)